src/types.ts

```diff
@@ -10,6 +10,8 @@ export type ChatGPTAPIOptions = {
   /** @defaultValue `'https://api.openai.com'` **/
   apiBaseUrl?: string
 
+  apiOrg: string
```

Comment:
@zhengxs2018 @transitive-bullshit Why it's not optional? Not it requires me to pass it to the constructor

Reply:
ahhh will fix this now; def a bug

Reply:
should be fixed now @alxmiron https://github.com/transitive-bullshit/chatgpt-api/releases/tag/v5.2.2

Reply:
@alxmiron  Sorry, It should be optional.